Error: Version "20.04" of Ubuntu is not supported

##[error]Unexpected error, unable to continue. Please report at https://github.com/fwal/setup-swift/issues
Version "20.04" of Ubuntu is not supported
Stacktrace:
Error: Version "20.04" of Ubuntu is not supported
    at Object.<anonymous> (/home/runner/work/_actions/fwal/setup-swift/v1.0.1/dist/index.js:7541:19)
    at Generator.next (<anonymous>)
    at fulfilled (/home/runner/work/_actions/fwal/setup-swift/v1.0.1/dist/index.js:7494:58)

--- a/src/os.ts
+++ b/src/os.ts
@@ -6,8 +6,8 @@ export enum OS {
 }
 
 const AVAILABLE_OS: { [platform: string]: string[] } = {
-  macOS: ["latest"],
-  Ubuntu: ["18.04", "16.04"],
+  macOS: ["latest", "11.0", "10.15"],
+  Ubuntu: ["latest", "20.04", "18.04", "16.04"],
 };
 
 export interface System {

